chart/jetstack-secure-gcm/charts/google-cas-issuer/values.yaml

```diff
@@ -26,10 +26,7 @@ prometheus:
   # Enables the creation of the ClusterIP service.
   enabled: true
 
-resources:
-  limits:
-    cpu: 200m
-    memory: 200Mi
-  requests:
-    cpu: 200m
-    memory: 200Mi
+resources: {}
+  # requests:
+  #   cpu: 10m
+  #   memory: 32Mi
```

Comment:
Might be worth having a look at how we are supposed to be setting the `request` and `limit` fields. I had to remove the resource request for the google-cas-issuer due to the smoke-test clsuter single node being too small, but I don't really know how we should do things with regards to the Marketplace UI

Reply:
cc @jakexks @irbekrm